db/funds.go

```diff
@@ -0,0 +1,93 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/filecoin-project/go-state-types/big"
+
+	"github.com/filecoin-project/go-state-types/abi"
+
+	"github.com/google/uuid"
+)
+
+type FundsLog struct {
+	DealUuid  uuid.UUID
+	CreatedAt time.Time
+	Text      string
+}
+
+type FundsDB struct {
+	db *sql.DB
+}
+
+func NewFundsDB(db *sql.DB) *FundsDB {
+	return &FundsDB{db: db}
+}
+
+func (f *FundsDB) Tag(ctx context.Context, dealUuid uuid.UUID, collateral abi.TokenAmount, pubMsg abi.TokenAmount) interface{} {
+	qry := "INSERT INTO FundsTagged (DealID, CreatedAt, Collateral, PubMsg) "
```

Comment:
Nit, but we might want to rename the field from `DealID` to `DealUUID` just so that we avoid confusion with the DealID on chain.

Reply:
I think it would be better to call it DealUUID 👍 
We're using DealID in other places in the DB, so I'm going to merge this code for consistency then open a follow-up PR to change the name everywhere

Reply:
https://github.com/filecoin-project/boost/issues/87